test/integration/scheduler/scheduler_test.go

```diff
@@ -162,39 +162,31 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		gomega.Expect(createdProdJob2.Spec.Template.Spec.NodeSelector[instanceKey]).Should(gomega.Equal(onDemandFlavor.Name))
 	})
 
-	ginkgo.It("Should schedule jobs on tolerated flavors", func() {
-		ginkgo.By("checking a job without toleration starts on the non-tainted flavor")
-		job1 := testing.MakeJob("on-demand-job1", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "5").Obj()
-		gomega.Expect(k8sClient.Create(ctx, job1)).Should(gomega.Succeed())
-		createdJob1 := &batchv1.Job{}
-		gomega.Eventually(func() *bool {
-			lookupKey := types.NamespacedName{Name: job1.Name, Namespace: job1.Namespace}
-			gomega.Expect(k8sClient.Get(ctx, lookupKey, createdJob1)).Should(gomega.Succeed())
-			return createdJob1.Spec.Suspend
-		}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(false)))
-		gomega.Expect(createdJob1.Spec.Template.Spec.NodeSelector[instanceKey]).Should(gomega.Equal(onDemandFlavor.Name))
+	ginkgo.It("Should schedule workloads on tolerated flavors", func() {
+		ginkgo.By("checking a workload without toleration starts on the non-tainted flavor")
+		wl1 := testing.MakeWorkload("on-demand-wl1", ns.Name).Queue(prodQueue.Name).Request(corev1.ResourceCPU, "5").Obj()
```

Comment:
Shouldn't this be probBEQueue?

Reply:
There is no probBEQueue now. prodQueue is changed to `BestEffortFIFO`.